projects/components/src/splitter/splitter.component.ts

```diff
@@ -44,7 +44,9 @@ import { debounce } from 'lodash-es';
           [ngStyle]="this.splitterSizeStyle"
           (mousedown)="this.onGutterMouseDown($event, index)"
         >
-          <div class="cursor"></div>
+          <div class="background">
+            <div class="cursor"></div>
```

Comment:
are you planning to add something else here or just you add it to update the styles of the container?